app/frontend/wallet/backend-types.ts

```diff
@@ -105,8 +88,15 @@ export type PoolRecommendationResponse = StakePoolInfoExtended & {
   isInRecommendedPoolSet: boolean
 }
 
+export type TokenObject = {
+  policyId: string
+  assetName: string
+  quantity: string // possibly huge amounts
```

Comment:
I would add a TODO and backlog issue to transition to BigInt (probably not the native one as Safari added it only recently) - anybody can easily mint/have tokens with huge amounts well beyond JS max safe int

Reply:
I agree, I intentionally dropped the support for bigints to not support them in some places, and don't support them in other places, we should handle the migration to bigint in separate issue dedicate to that so nothing is misses. I ll create an issue for it.